Training1/training/src/Questions.js

```diff
@@ -0,0 +1,9 @@
+export default function Questions(props) {
+  return (
+    <div>
+      <p>Question {props.index}</p>
```

Comment:
We should group the appropriate props

eg:
const { question, index } = props;

question.q
question.a